GOVERNANCE.md

```diff
@@ -0,0 +1,21 @@
+# Reference Types Working Group Governance
+
+## Discussion
+
+Discussions take place on GitHub's [discussion board](https://github.com/opencontainers/wg-reference-types/discussions) and during [weekly meetings](https://github.com/opencontainers/wg-reference-types/blob/main/README.md#meetings). Notes are taken down at the meetings are recorded in [this google document](https://docs.google.com/document/d/1SVOWQTowigXzbYdorzfa7tMmrcm91yK12LvSONqziJY/edit). Initial seeding of the GitHub discussion topics take place in [this google document](https://docs.google.com/document/d/1ky0LMAvWL_XRnfId9h5GBswV7GGko5GejuyaMWCGX-Y/edit#). It is expected that these google documents will be deprecated in favor of the GitHub Discussion board and GitHub issues.
+
+Discussions shall follow the [neutral framework](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md) previously decided by this working group. Changes to the framework are encouraged via GitHub discussions, issues, and pull requests.
```

Comment:
Is `neutral framework` is the best word to describe what this is? I think `documented personas` is more accurate.